datafusion/datasource-parquet/src/source.rs

```diff
@@ -343,9 +343,7 @@ impl ParquetSource {
     }
 
     /// If true, the predicate will be used during the parquet scan.
-    /// Defaults to false
-    ///
-    /// [`Expr`]: datafusion_expr::Expr
+    /// Defaults to false.
```

Comment:
How did this result in wrong results? 

Is it because the planner assumed filters would be pushed down but they weren't?

Reply:
Well this bit is just a comment 😛 

But this produced wrong results because we'd return `Supported(filter)` up to the parents -> `FilterExec` removed from the plan but `ParquetOpener.pushdown_filters = false` -> pruning did not actually happen during the scan.